### PR TITLE
Fixes typo and style errors : issue #3011

### PR DESCRIPTION
**Pull request by barun511**
## Status

Ready for review / Work in progress

## Description of Changes

Fixes #.

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the app code:

- [ ] Unit and functional tests pass on the development VM

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made changes to documentation:

- [ ] Doc linting passed locally

### CODE REVIEW

File: docs/test_the_installation.rst [hunk 1]
```diff
@@ -71,13 +71,13 @@ Test the web interfaces
...
-#. Test that you can access the Journalist Interface, and that you can log
+#. Test that you can access the *Journalist Interface*, and that you can log
    in as the admin user you just created.
 
    - Open the Tor Browser and navigate to the onion URL from
-     app-journalist-aths. Enter your passphrase and two-factor
+     ``app-journalist-aths``. Enter your passphrase and two-factor
      authentication code to log in.
-   - If you have problems logging in to the Admin/Journalist Interface,
+   - If you have problems logging in to the *Admin/Journalist Interface*,
```
Comment (heartsucker) [inline docs/test_the_installation.rst]: I think this is the only occurrence of "Admin Interface", so I think it should be removed to just say "Journalist Interface."

File: docs/test_the_installation.rst [hunk 2]
```diff
@@ -92,17 +92,17 @@ Test the web interfaces
...
 
-   - Within Tor Browser, navigate back to the app-source-ths URL and
+   - Within Tor Browser, navigate back to the ``app-source-ths`` URL and
      use your previous test source codename to log in (or reload the
      page if it's still open) and check that the reply you just made
      is present.
 
 #. We highly recommend that you create persistent bookmarks for the
-   Source and Journalist Interface addresses within Tor Browser.
+   *Source and Journalist* Interface addresses within Tor Browser.
```
Comment (heartsucker) [inline docs/test_the_installation.rst]: I think this should say `*Source Interface* and *Journalist Interface* addresses` for consistency.

**Comment by heartsucker** [comment]
Currently you're getting errors in CI because of this: https://github.com/freedomofpress/securedrop/issues/3049#issuecomment-367387722

Can you rebase on develop to ensure that the tests pass?

**Comment by dachary** [comment]
@barun511 rebased & repushed for your benefit, sorry for the inconvenience.

**Comment by dachary** [comment]
@heartsucker are these changes good for you ?

File: docs/test_the_installation.rst (hunk 1 above)
Comment (heartsucker) [reply docs/test_the_installation.rst]: Updating this comment to say that the docs use the term `Admin Interface` often enough that this wouldn't appear out of place.

**Comment by heartsucker** [comment]
@dachary I have one comment I would still like addressed.

**Comment by barun511** [comment]
While I'd like to address the given comment, I'm not entirely sure what to do after the rebase and repush. Do I simply add another commit to the branch I was working on and push again?

**Comment by b-meson** [comment]
Hi @barun511. If you are unfamiliar with rebasing in general, it generally takes this form. You would add an additional commit on your current working branch and then you would `git rebase -i <remote> <branch>`. Usually that means the remote branch is the securedrop repository and on top of the `develop` branch. Usually the command I run is `git rebase -i upstream develop` which assumed I have an `upstream` branch added as remote. This matches when I do `git remote -v` on the securedrop repo. 

Still confused? Join our [gitter](https://gitter.im/freedomofpress/securedrop) channel for help.

**Comment by dachary** [comment]
@barun511 ping?

**Comment by barun511** [comment]
Ping! I'm so sorry, I was in the middle of my midterm exams and forgot
about this. I'll get to it in an hour or so.

**Comment by barun511** [comment]
I got a little bit confused about the whole matter, so I created a new pull request that takes heartsucker's comments into account.

**Comment by heartsucker** [comment]
Linking current and new PR (#3147)

**Comment by heartsucker** [comment]
@barun511 Since it's better to rebase the current branch and squash commits than to open a new PR (it keeps the discussion in the same place), you should read [this section](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) from the git book. The command `git rebase -i` is usually what you need to clean up commits while you're making PRs.

**Comment by dachary** [comment]
Closing bcause it is a duplicate of https://github.com/freedomofpress/securedrop/pull/3147 and it lost the original branch because it was assigned to 3147. It would be better to keep this one and preserve the conversation but it is no longer possible.

**Pull request closed without merging**